README.md

```diff
@@ -1,47 +1,185 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
```

Comment:
Primary reason is because JSX doesn't work well in coffeescript/livescript because of its need to be compiled. Projects like cjsx have failed because lack of tooling support. This library works well for non-babel react codebases.